### PR TITLE
change prompt for enabling RBAC on local-up-cluster

### PR DESCRIPTION
**Pull request by xilabao**
run `ALLOW_ANY_TOKEN=true ENABLE_RBAC=true hack/local-up-cluster.sh`
Use "--token" instead of "--username/--password", so need to update the prompt

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35962)

### CODE REVIEW

**Comment by liggitt** [comment]
might need to harmonize with #36134

**Comment by xilabao** [comment]
harmonized with #36134

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -599,14 +599,24 @@ elif [[ "${START_MODE}" == "nokubelet" ]]; then
 fi
 
 if [[ "${START_MODE}" != "kubeletonly" ]]; then
+  auth_arg=""
+  if [[ -n "${ALLOW_ANY_TOKEN:-}"  ]]; then
+      if [[ -n "${KUBECONFIG_TOKEN:-}"  ]]; then
+          auth_arg=${KUBECONFIG_TOKEN}
+      else
+          auth_arg="--token=admin/system:masters"
```
Comment (deads2k) [inline hack/local-up-cluster.sh]: How about `system:admin/system:masters`?  When we start creating client certificates for core system roles, I think we'll continue the pattern of `system` prefixes denoting managed/curated things.

**Comment by deads2k** [comment]
Minor comment on the name, but otherwise good.  Update and squash?

**Comment by deads2k** [comment]
lgtm

File: hack/local-up-cluster.sh [hunk 2]
```diff
@@ -326,11 +332,11 @@ function start_apiserver {
     # which should be able to be used as the CA to verify itself
 
     anytoken_arg=""
-    if [[ -n "${ALLOW_ANY_TOKEN:-}" ]]; then
+    if [[ "${ALLOW_ANY_TOKEN}" = true ]]; then
       anytoken_arg="--insecure-allow-any-token "
     fi
     authorizer_arg=""
-    if [[ -n "${ENABLE_RBAC:-}" ]]; then
+    if [[ "${ENABLE_RBAC}" = true ]]; then
```
Comment (liggitt) [inline hack/local-up-cluster.sh]: We don't require exact match to `true` for other `ALLOW_*/ENABLE_*` options, why these?

Comment (deads2k) [inline hack/local-up-cluster.sh]: > liggitt 2 minutes ago Kubernetes member
> We don't require exact match to true for other ALLOW_*/ENABLE_* options, why these?

I'd claim the others should probably be updated.

Comment (liggitt) [inline hack/local-up-cluster.sh]: What's the benefit to treating ENABLE_RBAC=1 as false?

Comment (deads2k) [inline hack/local-up-cluster.sh]: > What's the benefit to treating ENABLE_RBAC=1 as false?

About the same reasoning on as `--bool-flag=1` doesn't work.

**Comment by xilabao** [comment]
@k8s-bot unit test this

**Comment by xilabao** [comment]
@k8s-bot unit test this

**Comment by xilabao** [comment]
@k8s-bot unit test this

**Comment by deads2k** [comment]
@k8s-bot unit test this: issue #32988

**Comment by xilabao** [comment]
@deads2k I add the default set at line 337 when using ALLOW_ANY_TOKEN,  PTAL. thanks

**Comment by xilabao** [comment]
fix conflict， @deads2k PTAL

**Comment by deads2k** [comment]
tests don't cover this script with these options anyway

**Pull request merged**